polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -1065,12 +1068,19 @@ private class BasePolarisCatalogViewBuilder extends BaseMetastoreViewCatalog.Bas
 
     public BasePolarisCatalogViewBuilder(TableIdentifier identifier) {
       super(identifier);
+      withProperties(viewDefaultProperties());
```

Comment:
If Iceberg would have options to add catalog level properties for view like table. This is not required. I have created a [PR](https://github.com/apache/iceberg/pull/11025) with iceberg to support catalog level properties. 

Or are there any other ways to pass catalog level properties to views ?